docs/changelog/6_x.rst

```diff
@@ -0,0 +1,219 @@
+v6.0
+====
+
+:edb-alt-title: EdgeDB v6
+
+To explore the new features, ensure you specify version 6.0 when initializing
+your project. Pre-release versions are not considered stable and will not be
+automatically suggested:
+
+.. code-block:: bash
+
+  $ edgedb project init --server-version 6.0-beta.1
+
+
+Upgrading
+=========
```

Comment:
I'd collapse this section, we have an example somewhere

Reply:
Just during the beta period?